events/cmd/root.go

```diff
@@ -27,7 +27,9 @@ func EventCmd() *cobra.Command {
 	}
 
 	cmd.Flags().String(flags.FlagNode, "tcp://localhost:26657", "The node address")
-	_ = viper.BindPFlag(flags.FlagNode, cmd.Flags().Lookup(flags.FlagNode))
+	if err := viper.BindPFlag(flags.FlagNode, cmd.Flags().Lookup(flags.FlagNode)); err != nil {
+		return nil
```

Comment:
does caller check for nil result from EventCmd?

Reply:
A `nil` will result in the app not starting at all. Errors here only happen when adding conflicting flags. IMO this is sufficent.